static/app/views/issueDetails/streamline/eventNavigation.tsx

```diff
@@ -342,8 +342,14 @@ function EventNavigationLink({
     config?.initialCollapse ?? false
   );
   return (
-    <Button
-      onClick={() => {
+    <LinkButton
+      to={{
+        ...location,
+        hash: `#${config.key}`,
+        replace: true,
```

Comment:
is replace the right choice

Reply:
I don't mind it, but I don't think the other one replaced did it?